airflow-core.yaml

```diff
@@ -0,0 +1,136 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - coreutils
+      - curl
+      - cyrus-sasl
+      - cyrus-sasl-dev
+      - findutils
+      - glibc-locale-en
+      - glibc-locales
+      - gnupg-utils
+      - gzip
+      - mariadb-dev
+      - mysql
+      - openssl
+      - postgresql-client
+      - postgresql-dev
+      - python-3.12
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - gcc
+      - glibc-dev
+      - mariadb-connector-c-dev
+      - mariadb-dev
+      - openssf-compiler-options
+      - pkgconf-dev
+      - postgresql-dev
+      - py3.12-build-bin
+      - py3.12-pip
+      - python-3.12
+      - python-3.12-dev
+      - wolfi-base
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/airflow
+      tag: ${{package.version}}
+      expected-commit: c083e456fa02c6cb32cdbe0c9ed3c3b2380beccd
+
+  - runs: |
+      # by default airflow celery provider is not built,but running the upstream helm chart requires it
+       python patch.py
+       # remove the patcher
+       rm -rf patch.py
+
+  - runs: |
+      # Set required build environment variables
+      export SOURCE_DATE_EPOCH=315532800
+      export MYSQLCLIENT_CFLAGS=$(mysql_config --cflags)
+
+      # Install airflow core, the upstream sets this as directory
+      export PYTHONUSERBASE=${{targets.destdir}}/home/airflow/.local
+      pip3.12 install --no-cache-dir --user apache-airflow[core]==${{package.version}}
```

Comment:
Do we need to specify 'pip3.12' here or can we just use 'pip' given we've only one version installed? The latter would make maintaining this package easier in future, and less to update when we bump the python version.

Similar for the references below

Reply:
Thanks Mark, yes I don't think that should be a problem, i though to have it explicitly 3.12 can help know that we are using that as that is a requirement by upstream but having it in environment should be enough too